docker/datahub-gms/env/docker.postgres.env

```diff
@@ -1,25 +1,7 @@
-DATAHUB_UPGRADE_HISTORY_KAFKA_CONSUMER_GROUP_ID=generic-duhe-consumer-job-client-gms
```

Comment:
This file is now meant to be used in conjunction with docker.env, rather than instead of it